src/commonMain/kotlin/com/github/quillraven/fleks/family.kt

```diff
@@ -163,7 +165,7 @@ data class Family(
     /**
      * Returns true if and only if the given [entity] is part of the family.
      */
-    operator fun contains(entity: Entity): Boolean = entityBits[entity.id]
+    operator fun contains(entity: Entity): Boolean = !privateEntities.hasNoValueAtIndex(entity.id)
```

Comment:
Add a new `hasValue` or use `getOrNull(entity.id) != null`. Double negation is always confusing imo :)